shadowsocks-csharp/View/OnlineConfigForm.Designer.cs

```diff
@@ -213,6 +213,6 @@ private void InitializeComponent()
         private System.Windows.Forms.Button DeleteButton;
         private System.Windows.Forms.Button OkButton;
         private System.Windows.Forms.Button UpdateAllButton;
-        private System.Windows.Forms.Button CancelButton;
+        private new System.Windows.Forms.Button CancelButton;
```

Comment:
🤔

Reply:
This is recommended by IntelliSense to override the Forms CancelButton.